src/store/reducers/storage/storage.js

```diff
@@ -335,19 +335,6 @@ export const getFlatListStorage = createSelector(
     },
 );
 
-export const getVisibleEntitiesList = createSelector(
-    [getVisibleEntities, getFlatListStorage],
-    (visibleGroups, storageList) => {
-        if (visibleGroups === VISIBLE_ENTITIES.all) {
-            return storageList;
-        } else if (visibleGroups === VISIBLE_ENTITIES.missing) {
-            return _.filter(storageList, (g) => g.Missing > 0);
-        } else {
-            return _.filter(storageList, (g) => g.UsedSpaceFlag > 100);
-        }
-    },
-);
-
```

Comment:
It should be filtered on backend. Additionally, currently it's incorrect, that causes an issue, where no out of space nodes are displayed